### PR TITLE
Add document label check robot

### PR DESCRIPTION
**Pull request by fengtao1998**
### Motivation
To improve doc workflow efficiency, we want to introduce Bot to Pulsar community

### Modifications

1.Modified pull request template

2.Added YML file for BOT running

### Documentation

Check the box below and label this PR (if you have committer privilege).

Need to update docs? 

- [ ] `doc-required` 
  
  (If you need help on updating docs, create a doc issue)
  
- [ ] `no-need-doc` 
  
  (Please explain why)
  
- [ ] `doc` 
  
  (If this PR contains doc changes)

You can see the details of BOT here(https://docs.google.com/document/d/1bQfZkSu5nG1tNycpmXXtUFn-Z5-h-uqHv6IXsCEySQ8/edit#)

You can try to test docbot in this repo(https://github.com/fengtao1998/cherry_test)

If you need more BOT functions, please comment

### CODE REVIEW

**Comment by Anonymitaet** [comment]
@timmyyuan @codelipenghui could you please help review? Many thanks!

File: .github/workflows/ci-documentbot.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+# this action on a certain interval.And check the label about the
+# document.
+
+jobs:
+  labeling:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: pr-bot-test/auto-labeling@master
```
Comment (eolivelli) [inline .github/workflows/ci-documentbot.yml:24]: I believe that is it not allowed in the ASF to use third party GH Actions using branch name references.
Can we copy this code into an ASF repository ? or at least point to a specific SHA (if it is possible)

**Review by eolivelli — CHANGES_REQUESTED** [review]

File: .github/workflows/ci-documentbot.yml (hunk 1 above)
Comment (timmyyuan) [reply .github/workflows/ci-documentbot.yml:24]: Can we put this action into https://github.com/apache/pulsar-test-infra ?

Comment (fengtao1998) [reply .github/workflows/ci-documentbot.yml]: > I believe that is it not allowed in the ASF to use third party GH Actions using branch name references.
> Can we copy this code into an ASF repository ? or at least point to a specific SHA (if it is possible)

I have fixed the problem of GH action using third-party branches. I changed to point a sha to quote script. Do you think this is OK? Or can you tell me where the ASF repo is? I'll push the script in.

Comment (eolivelli) [reply .github/workflows/ci-documentbot.yml]: this is the repo
https://github.com/apache/pulsar-test-infra

having it in our repo is better, as private repos may disappear some day

Comment (fengtao1998) [reply .github/workflows/ci-documentbot.yml]: Thank you. I have submitted a pullrequest in this repo. When the merge succeeds, I will apply for review again.

Comment (fengtao1998) [reply .github/workflows/ci-documentbot.yml]: I have changed the repo in this GH Actions. 
PTAL, thanks.

**Review by eolivelli — APPROVED** [review]
looks great

**Pull request merged**